core/lib/components/balena/sdk.js

```diff
@@ -65,12 +65,17 @@ module.exports = class BalenaSDK {
 	constructor(
 		apiUrl,
```

Comment:
Since we will be changing argument, let's drop this one and fetch it from config.

Reply:
@vipulgupta2048 I still prefer giving the arguments insatead of a big blob of config

Reply:
this url is retrieved from the `config.js` anyway, it is then jsut passed to the cloud object via an argument

Reply:
Agreed and also agreed on providing backwards compatibility too, this will at least not break current test ™️